docs/getting-started/install/index.md

```diff
@@ -36,4 +36,24 @@ $ sudo yum install shopify-cli-x.y.x.rpm
 
 ```console
 $ gem install shopify-cli
-```
\ No newline at end of file
+```
+
+### To check that Shopify App CLI is installed correctly:
+
+```console
+$ shopify version
+1.0.0
+$
+```
+
+> Note that the version displayed may be newer.
+
+If the `shopify version` command above doesn't display a version number but instead shows:
+
+```console
+$ shopify version
+shopify command is no longer bundled with shopify_api.
+if you need these tools, install the shopify_cli gem
+$
+```
+then you will also need to upgrade the `shopify_api` gem to v9.2.0+, in order to remove a deprecated `shopify` command that is contained in that gem.
```

Comment:
Feels a little awkward to have a multiline code block plopped in the middle of a sentence. Suggest refactoring it so it's more like:

> If you have the `shopify_api` gem installed on your system, you might see this error:
> ```
> {error text}
> ```
> If you get this error after migrating, upgrade the `shopify_api` gem to v9.2.0 by running `whatever`...